qiskit_machine_learning/algorithms/classifiers/__init__.py

```diff
@@ -1,6 +1,6 @@
 # This code is part of Qiskit.
 #
-# (C) Copyright IBM 2020, 2021.
+# (C) Copyright IBM 2020, 2022.
```

Comment:
Well, it was a suggestion by the copyright tool.

Reply:
Strange, is this after running it locally from the recent capability to auto-fix the dates? If so @manoelmarques any thoughts. Otherwise the last time I can see it was updated was Dec last year so the 2021 would otherwise seem correct.

Reply:
Reverted back, let see what CI tells us.